.github/workflows/ci.yml

```diff
@@ -231,3 +265,31 @@ jobs:
     - name: Run notebooks 2
       timeout-minutes: 15
       run: tox -e notebooks2
+
+  notebooks3:
```

Comment:
this seems to share many dependencies with notebook2, so maybe just add this nb there? it seems to take not too much time to run the new notebook.

Reply:
I expect that additional notebooks for model selection may be added, to demonstrate new features as they are added to PEtab Select, which is why `notebooks3` is described as for model selection in `run_notebook.sh`. But could also merge with `notebooks2` for now.

Reply:
maybe merge atm I think. there is some limit on concurrent jobs in gha, so as long as the installation time outweighs the notebook runtime, I'd say try adding to the other one. easy to split when more notebooks are added.